firmware/biome.json

```
@@ -1,26 +1,27 @@
     "enabled": false,
     "clientKind": "git",
     "useIgnoreFile": false
   },
   "files": {
-    "include": ["stackchan", "mods", "tests", "scripts"],
-    "ignoreUnknown": false,
-    "ignore": [
-      "stackchan/speeches/calculate-power.js",
-      "stackchan/utilities/esp32/mac-address.js"
-    ]
+    "includes": [
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)